spec/unit/who_dis_syntax_error_spec.rb

```diff
@@ -82,27 +82,45 @@ class Blerg
       end
     end
 
-    it "determines the type of syntax error to be an unmatched bracket" do
-      source = <<~EOM
-        module Hey
-          class Foo
-            def initialize
-              [1,2,3
-            end
+    context "determines the type of syntax error to be an unmatched bracket" do
+      it "with missing bracket" do
+        source = <<~EOM
+          module Hey
+            class Foo
+              def initialize
+                [1,2,3
+              end
 
-            def call
+              def call
+              end
             end
           end
-        end
-      EOM
+        EOM
+
+        expect(
+          DeadEnd.invalid_type(source).error_symbol
+        ).to eq(:unmatched_syntax)
+
+        expect(
+          DeadEnd.invalid_type(source).unmatched_symbol
+        ).to eq(:"[")
```

Comment:
@schneems I realized this was wrong even in the previous commit, now it shows up correctly "reversed", as shown in https://github.com/zombocom/dead_end/pull/89/files#r731862726, shows the symbol that failed to match, and what the missing character could be.